cmd/conformance-tester/pkg/scenarios/anexia.go

```diff
@@ -101,7 +101,31 @@ func (s *anexiaScenario) NodeDeployments(_ context.Context, num int, secrets typ
 				Template: &apimodels.NodeSpec{
 					Cloud: &apimodels.NodeCloudSpec{
 						Anexia: &apimodels.AnexiaNodeSpec{
-							DiskSize:   pointer.Int64(nodeDiskSize),
+							DiskSize:   nodeDiskSize,
+							CPUs:       pointer.Int64(nodeCpu),
+							Memory:     pointer.Int64(nodeMemory),
+							TemplateID: &secrets.Anexia.TemplateID,
+							VlanID:     &secrets.Anexia.VlanID,
+						},
+					},
+					Versions: &apimodels.NodeVersionInfo{
+						Kubelet: s.version.String(),
+					},
+					OperatingSystem: osSpec,
+				},
+			},
+		},
+		{
```

Comment:
have a hard time reading the prow log, is this doing what I want?

Create an additional machine deployment, this time with the new Disks attribute instead of the old DiskSize attribute. Having both MachineDeployments around, makes sure in E2E testing, that both variants work as expected.

Reply:
Yeah, that seems right.